### DISCUSSION TITLE
I really need sd-webui-supermerger and sd-webui-untitledmerger to work on ForgeUI

### DISCUSSION THREAD

**Discussion by DucHaiten** [lllyasviel/stable-diffusion-webui-forge#2487] [General]
please someone in the community, I really need sd-webui-supermerger and sd-webui-untitledmerger to work on ForgeUI. I really like ForgeUI but many good Extensions only work on automatic1111

**Comment by SvenErik1968** [comment]
I think your best chance for this to happen is to contact the developers of those extensions and ask if they are willing to make them compatible with Forge.
